tools/reckless

```diff
@@ -983,6 +986,37 @@ def cargo_installation(cloned_plugin: InstInfo):
     return cloned_plugin
 
 
+def install_python_uv(cloned_plugin: InstInfo):
```

Comment:
why not just replace pip installation with `uv` as it supports all its capabilities

Reply:
> why not just replace pip installation with uv as it supports all its capabilities 

What if user doesn't have uv but has pip/poetry?

Reply:
Makes sense yeah I got the answer myself after pondering on it just didn't get to reply. We should definitely keep pip but remove poetry in the later releases probably as giving people too many options to do a single thing doesn't seem like a good idea.

If we can just pivot on `uv` as primary python plugin management tool that would be great.